java/connector-node/risingwave-connector-test/src/test/java/com/risingwave/connector/source/PostgresSourceTest.java

```diff
@@ -284,4 +284,76 @@ public void getTestJson() throws InterruptedException, SQLException {
         Thread.sleep(1000);
         connection.close();
     }
+
+    @Ignore
+    @Test
+    public void getTestJsonDateTime() throws InterruptedException, SQLException {
+        Connection connection = SourceTestClient.connect(pgDataSource);
+        String query =
+                "CREATE TABLE orders ("
+                        + "o_key integer, "
+                        + "o_time_0 time(0), "
+                        + "o_time_6 time(6), "
+                        + "o_timez_0 time(0) with time zone, "
+                        + "o_timez_6 time(6) with time zone, "
+                        + "o_timestamp_0 timestamp(0), "
+                        + "o_timestamp_6 timestamp(6), "
+                        + "o_timestampz_0 timestamp(0) with time zone, "
+                        + "o_timestampz_6 timestamp(6) with time zone, "
+                        + "o_interval interval, "
+                        + "o_date date, "
+                        + "PRIMARY KEY (o_key))";
+        SourceTestClient.performQuery(connection, query);
+        query =
+                "INSERT INTO orders VALUES (0, '11:11:11', '11:11:11.00001', '11:11:11Z', '11:11:11.00001Z', '2011-11-11 11:11:11', '2011-11-11 11:11:11.123456', '2011-11-11 11:11:11',  '2011-11-11 11:11:11.123456', '01:00:00', '1999-09-09')";
+        SourceTestClient.performQuery(connection, query);
+        Iterator<ConnectorServiceProto.GetEventStreamResponse> eventStream =
+                testClient.getEventStreamStart(
+                        pg, ConnectorServiceProto.SourceType.POSTGRES, "test", "orders");
+        Thread t1 =
```

Comment:
You should use an executor service rather than use thread for async task

Reply:
ohh..